.rubocop_todo.yml

```diff
@@ -1,44 +1,110 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2016-10-28 16:34:29 +0300 using RuboCop version 0.44.1.
-
+# on 2018-03-20 17:48:28 +0000 using RuboCop version 0.53.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
-Metrics/AbcSize:
-  Max: 16
+# Offense count: 2
+# Cop supports --auto-correct.
+# Configuration parameters: EnforcedStyleAlignWith, AutoCorrect, Severity.
+# SupportedStylesAlignWith: keyword, variable, start_of_line
+Layout/EndAlignment:
   Exclude:
     - 'lib/ossert/fetch/github.rb'
 
+# Offense count: 2
+# Configuration parameters: AllowSafeAssignment.
+Lint/AssignmentInCondition:
+  Exclude:
+    - 'lib/ossert/fetch/stackoverflow.rb'
+    - 'lib/ossert/workers/partial_fetch.rb'
+
+# Offense count: 2
+Lint/DuplicateMethods:
+  Exclude:
+    - 'lib/ossert/classifiers/decision_tree.rb'
+    - 'lib/ossert/quarters_store.rb'
+
+# Offense count: 1
+# Configuration parameters: IgnoreImplicitReferences.
+Lint/ShadowedArgument:
+  Exclude:
+    - 'lib/ossert/repositories.rb'
+
+# Offense count: 27
+Metrics/AbcSize:
+  Max: 150
```

Comment:
Was 16 but became 150... Maybe we could Exclude something here and restore previous restriction?